src/wizards/modify_task.xml

```diff
@@ -156,6 +162,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             </xsl:if>
 
             <xsl:if test="(/wizard/params/param[name='reschedule']/value = 1) and (/wizard/params/param[name='start_year']/value != '') and (/wizard/params/param[name='start_month']/value != '') and (/wizard/params/param[name='start_day']/value != '') and (/wizard/params/param[name='start_hour']/value != '') and (/wizard/params/param[name='start_minute']/value != '')">
+              <xsl:variable name="start">
+                <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_year']/value), 2)"/>
+                <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_month']/value), 2)"/>
+                <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_day']/value), 2)"/>
+                <xsl:text>T</xsl:text>
+                <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_hour']/value), 2)"/>
+                <xsl:value-of select="substring(string('100' + /wizard/params/param[name='start_minute']/value), 2)"/>
+                <xsl:text>00Z</xsl:text>
```

Comment:
I think users would expect the time to be entered as local time according the chosen timezone instead of UTC.

Reply:
The user actually provides the timezone.  Adjusted to use that timezone in 86c3adb.